app/presenters/publishing_api/contact_presenter.rb

```diff
@@ -71,34 +68,40 @@ def contact_type
     end
 
     def details
-      details = { description: description, contact_type: contact_type, title: title }
-      details[:contact_form_links] = [contact_form_links] if contact_form_url
-      details[:post_addresses] = [post_address] if postal_code
-      details[:email_addresses] = [email_address] if email
+      details = {
+        title: title,
+        description: comments.presence,
+        contact_type: contact_type,
+      }.compact
+
+      details[:contact_form_links] = [contact_form_links] if contact_form_url.present?
+      details[:post_addresses] = post_addresses
+      details[:email_addresses] = [email_address] if email.present?
       details[:phone_numbers] = phone_numbers if contact_numbers.any?
 
       details
     end
 
     def contact_form_links
       {
-        title: title,
```

Comment:
Just a quick note-to-self this isn't used by govspeak when the form link is rendered.